### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by Rockyraven**

### CODE REVIEW

File: component/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ :root{
+    --primary-color: #ff3b30;
+    --seconday-color: #282828;
+ }
+ .header-content{
+     
+     height: 5rem;
+     width: 100%;
+     background-color: var(--secondary-color);
+     /* display: inline; */
```
Comment (arjun-computer-geek) [inline component/alert/alert.css]: remove these commented or unused codes

File: component/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
+@import url('/style.css');
+ @import url( '/component/slidebaar.css');
+ :root{
+    --primary-color: #ff3b30;
```
Comment (arjun-computer-geek) [inline component/alert/alert.css]: use these color's variables in style.css . Thereafter you can use it anywhere.

File: component/alert/alert.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+     
+     height: 5rem;
+     width: 100%;
+     background-color: var(--secondary-color);
+     /* display: inline; */
+     /* display: flex; */
+    
+ }
+  header h1{
+     margin-top: 15px;
```
Comment (arjun-computer-geek) [inline component/alert/alert.css]: use rem instead px

File: component/alert/alert.css [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+     border-radius: 5px;
+     
+    }
+    .alert-warning{
+        background-color: rgb(211, 233, 132);
+        border-radius: 5px;
+        
+    }
+    .alert-error{
+        background-color: rgb(245, 102, 91);
```
Comment (arjun-computer-geek) [inline component/alert/alert.css]: hex code would be better

File: component/alert/alert.css [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    }
+    .alert-info{
+        background-color: rgb(132, 134, 233);
+        border-radius: 5px;
+        
+    }
+    p{
+     font-size: 1.5rem;
+ }
+ /* .all-containant{
```
Comment (arjun-computer-geek) [inline component/alert/alert.css]: remove these too

File: component/alert/alert.html [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="alert.css">
+</head>
+<script src="https://kit.fontawesome.com/a076d05399.js"></script>
```
Comment (arjun-computer-geek) [inline component/alert/alert.html]: write comments. it would be better for others

File: component/slidebaar.css [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+}
+.sidebar{
+    height: 100%;
+    
+    width: 20%;
+     display: fixed;
+   
+    
+}
+
```
Comment (arjun-computer-geek) [inline component/slidebaar.css]: why this extra spacing? Any reason?

File: index.html [hunk 8]
```diff
@@ -23,7 +23,7 @@ <h1 class="heading">BOOSTWEB UI</h1>
         </header>
         <div class="description-part">
             <h2>Welcome to <span> BOOSTWEB UL</span> world</h2>
-            <a href="#">Explore More</a>
+            <a href="http://127.0.0.1:5500/alert.html">Explore More</a>
```
Comment (arjun-computer-geek) [inline index.html]: don't add localhost references in href

**Review by arjun-computer-geek — CHANGES_REQUESTED** [review]

File: alert.html [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+    <link rel="stylesheet" href="alert.css">
+</head>
+<script src="https://kit.fontawesome.com/a076d05399.js"></script>
+<body>
+    <header>
+        <h1 class="heading">BOOSTWEB UI</h1>
+        <div class="social-media-link">
+            <a href="#"> <i class="fab fa-instagram" style="font-size:40px"></i> </a>
+            <a href="#"> <i class="fab fa-linkedin" style="font-size:40px"></i></a>
+            <a href="#"> <i class="fab fa-github" style="font-size:40px"> </i></a>
```
Comment (pratik-kumar-sahu) [inline alert.html]: try to minimize repetitive codes. Use classes instead of inline styling wherever possible.

**Review by pratik-kumar-sahu — COMMENTED** [review]
- you can use "Prettier" in vs code to get rid of uneven spaces in your code.
- give a deployed link also in your repo description for easy access.

Everything else is fine. Nice work.

File: style.css [hunk 10]
```diff
@@ -2,7 +2,7 @@ body{
     margin: 0px;
     padding: 0px;
     /* background-color: rebeccapurple; */
-    background-image: url(./component-wallpaper1.jpg);
+    /* background-image: url(./component-wallpaper1.jpg); */
 }
```
Comment (Shubhamkr2610) [inline style.css]: remove unused code

File: style.css [hunk 11]
```diff
@@ -27,7 +27,7 @@ flex-wrap: wrap;
...
 .social-media-link a{
     margin-right: 2.5rem;
-    color: white;
+    color: red;
 }
```
Comment (Shubhamkr2610) [inline style.css]: I think you should you color with variable name

File: component/alert/alert.css [hunk 12]
```diff
@@ -1,64 +1,54 @@
...
+
     .alert-warning{
         background-color: rgb(211, 233, 132);
-        border-radius: 5px;
-        
+        border-radius: 5px;    
     }
+
     .alert-error{
         background-color: rgb(245, 102, 91);
```
Comment (Shubhamkr2610) [inline component/alert/alert.css]: use color name with variable

**Pull request merged**